packages/fluentui/react-northstar/src/components/Avatar/AvatarImage.tsx

```diff
@@ -0,0 +1,120 @@
+import { Accessibility, AccessibilityAttributes, imageBehavior, ImageBehaviorProps } from '@fluentui/accessibility';
+import {
+  ComponentWithAs,
+  getElementType,
+  useUnhandledProps,
+  useAccessibility,
+  useFluentContext,
+  useStyles,
+  useTelemetry,
+} from '@fluentui/react-bindings';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+import { UIComponentProps, commonPropTypes } from '../../utils';
+import { FluentComponentStaticProps } from '../../types';
+import { imageClassName } from '../Image/Image';
+
+export interface AvatarImageProps extends UIComponentProps, ImageBehaviorProps {
```

Comment:
Can `AvatarImageProps` extend `ImageProps` ? is there any reason currently why they should be different ?

Reply:
Well, do this wouldn't be like coupling them?

Reply:
I agree it's not ideal, but by decoupling them we increase the risk of introducing an accidental breaking change.

So while it might not be a breaking change now, we might forget that the prop interface might be the same as `Image` in the future. And a subsequent modification can be a breaking change.

However a workaround could be a comment somewhere that if modifications happen to these props to verify that it diverges from `ImageProps`. If it diverges then it's a breaking change.

It's a hard one even by the current standards we track breaking changes, maybe another reviewer might see that it's pointless or have a better solution :-)

Reply:
It's same props as for `Image`, literally copy and paste